### PR TITLE
Note about reusing parameters in SelectQueryBuilder's docs

### PR DESCRIPTION
**Pull request by foxdb**
Documentation: actions discussion on: https://github.com/typeorm/typeorm/issues/3428

### CODE REVIEW

File: docs/select-query-builder.md [hunk 1]
```diff
@@ -274,6 +274,19 @@ is a shortcut for:
 .setParameter("name", "Timber")
 ```
 
+Note: a set of parameters is unique per-query. A parameter's value will be overriden if you set it multiple times:
```
Comment (pleerock) [inline docs/select-query-builder.md]: Thank you for your contribution! But I think we just need a simple one statement, like "note: do not use same parameter names for different values across your query builder" without code examples to simplify docs.

Comment (foxdb) [reply docs/select-query-builder.md]: Updated!

**Comment by pleerock** [comment]
Thank you!

**Comment by vtgn** [comment]
I totally disagree on this easy solution. It is really horrible and illogical! It would have been fixed in the code.

**Pull request merged**